### DISCUSSION TITLE
Dashboard: Remove surrounding `[ ]` from `key`?

### DISCUSSION THREAD

**Discussion by minusfive** [LazyVim/LazyVim#1746] [Q&A] [answered]
Any idea how one can remove the surrounding square brackets from shortcut `key`s on dashboard? The screenshots on https://github.com/nvimdev/dashboard-nvim don't seem to include it, nor am I able to find where in LazyVim's default config this might've been added. Any idea?

![Screenshot 2023-10-15 at 11 07 36 PM](https://github.com/LazyVim/LazyVim/assets/33695/d54162ed-18ac-47e4-afdf-c9faa61b7409)

**Comment by folke** [comment, marked as answer]
That's hard-coded in the `dashboard.nvim` plugin here https://github.com/nvimdev/dashboard-nvim/blob/fd68b9d05aedc57e5d97a6b66a4e3db877872f64/lua/dashboard/theme/doom.lua#L99

You can open an issue in the repo, or provide them with a PR to have that configurable.

When that's done, I'm open to a PR for LazyVim to remove the `[]`

**Comment by minusfive** [comment]
@folke submitted https://github.com/nvimdev/dashboard-nvim/pull/396 🤞

**Reply by folke** [reply]
Great!

**Reply by minusfive** [reply]
I know you already got notified, but adding it here for the discussion https://github.com/LazyVim/LazyVim/pull/1791

![Screenshot 2023-10-18 at 1 08 02 PM](https://github.com/LazyVim/LazyVim/assets/33695/17b784b4-6194-417c-9493-626c37e9c5dc)
